jmclient/test/test_coinjoin.py

```diff
@@ -119,15 +124,17 @@ def raise_exit(i):
     set_commitment_file(str(tmpdir.join('commitments.json')))
 
     MAKER_NUM = 3
-    wallets = make_wallets_to_list(make_wallets(
+    wss = make_wallets_to_list(make_wallets(
```

Comment:
this is missing the call to the previously defined but unused `wallets_to_wallet_services`

Reply:
I see `make_wallets` now returns `WalletService` objects instead and even saves them in the `wallet` key. That's confusing. `wallets_to_wallet_services` can be removed?

Reply:
Yes, exactly, sorry I just forgot to clean this up. Fixed in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/359/commits/e1fbc427120e31fdc3e96fe761720abf085390f2